tools/http3_test/src/runner.rs

```diff
@@ -140,6 +141,11 @@ pub fn run(
 
                 conn.on_timeout();
 
+                if conn.is_closed() && reqs_complete != reqs_count {
+                    panic!("Client timed out after {}ms and only completed {}/{} requests",
+                    IDLE_TIMEOUT, reqs_complete, reqs_count);
```

Comment:
Also, we already track the start of the connection in `req_start`, so maybe we can just do `req_start.elapsed()` which would be more precise than just using the value of the idle timeout.

Reply:
example `Client timed out after 60.236662187s` :D